code/__HELPERS/text.dm

```diff
@@ -305,6 +305,13 @@ proc/checkhtml(var/t)
 			return copytext(text, 1, i)
 	return text
 
+//Returns the last word in a string.
+/proc/get_last_word(text)
+	for(var/i = 1 to length(text))
+		if(text2ascii(text, length(text)-i) == 32)
+			return copytext(text, length(text)-i, length(text)+1)
```

Comment:
You should probably be using the much simpler `splittext()` implementation so make that change instead of this one, but for the record, this should be `copytext(text, -length(text))` as `copytext()` supports negative indices and the default end position of `0` is the same as `length(text)+1`